src/helpers.ts

```diff
@@ -33,3 +33,28 @@ export function chunkArray<T>(array: T[], size: number) {
   }
   return result;
 }
+
+export function oldApyToSnakeCase(apy: Apy | undefined): Apy | undefined {
```

Comment:
I think I'd echo the other comment about moving functionality into `src/services/vision.ts` here, seems like this logic is tightly coupled to that file.

Also, since we know the upstream exporter code, maybe it'd be more efficient to only apply this function to `Apy` objects which have the type `backscratcher`, then we can know we're ok to decode `composite` as `BackscracherApyComposite` and convert it to snake case.

Also I might be nitpicking here but maybe a better function name would be something like `convertCompositeApyToSnakeCase`

Reply:
Data from the vaults seem to come from https://api.yearn.finance/v1/chains/1/vaults/all and `vision.ts` is querying `https://d28fcsszptni1s.cloudfront.net/v1/chains/${this.chainId}/vaults/all` is it still correct to move this logic to vision.ts?

Reply:
Yeah it's the same thing, I can't remember why we use cloudfront directly but probably best to leave it like that